package.json

```diff
@@ -39,16 +39,23 @@
     "script"
   ],
   "dependencies": {
-    "each-async": "^1.1.1",
     "meow": "^3.0.0",
-    "osx-trash": "^1.0.1",
+    "osx-trash": "^2.0.0",
     "path-exists": "^1.0.0",
+    "pify": "^2.2.0",
+    "pinkie-promise": "^1.0.0",
     "update-notifier": "^0.3.0",
-    "win-trash": "^1.0.1",
-    "xdg-trash": "^1.0.0"
+    "win-trash": "^2.0.0",
+    "xdg-trash": "^2.0.1"
   },
   "devDependencies": {
-    "mocha": "*",
-    "path-exists": "^1.0.0"
+    "ava": "*",
+    "path-exists": "^1.0.0",
```

Comment:
Yeah, it gives you a good overview of what's needed for the tests to run. If you suddenly doesn't need `path-exists` anymore in your regular dependencies and you forget that your tests depends on it they will fail.